lib/auth/auth_test.go

```diff
@@ -1285,7 +1285,7 @@ func TestNewWebSession(t *testing.T) {
 	}
 	bearerTokenTTL := utils.MinTTL(req.SessionTTL, BearerTokenTTL)
 
-	ws, err := p.a.NewWebSession(req)
+	ws, err := p.a.NewWebSession(context.Background(), req)
```

Comment:
Reuse the context declared above?

```suggestion
	ws, err := p.a.NewWebSession(ctx, req)
```

Reply:
Did I look at the wrong place? Thanks for the changes anyway.